src/llfs/data_packer.hpp

```diff
@@ -137,10 +137,15 @@ class DataPacker
            (rec_end <= this->buffer_end());
   }
 
+  /*! \brief Allocate buffer from Data-packer's arena for the passed in Type.
+   *
+   * Param 'count' is specifying number of elements of type 'T' for which memory allocation is
+   * requested. By default it is going allocate space for 'one' element.
```

Comment:
nit: "going allocate" => "going to allocate"

Please use Doxygen tag: `\param count  The number of elements...`